pkgs/by-name/jo/journalist/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildGoModule,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "journalist";
+  version = "1.0.0-unstable-2024-05-28";
+
+  src = fetchFromGitHub {
+    owner = "mrusme";
+    repo = "journalist";
+    rev = "eda6ce67c975f928ba2dfe94a39b69d3f864d051";
+    hash = "sha256-pxWBvh7j+IRvGLpQA0TEqorw7f0OZtXux5svMMqVooA=";
+  };
+
+  overrideModAttrs = _oldAttrs: {
+    postPatch = ''
+      cp ${./go.sum} go.sum
+    '';
+  };
+
+  vendorHash = "sha256-fEHVc9kRbeeXICWhJshLp9JK/ICBR/RB5SVChJzSXpI=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/mrusme/journalist/journalistd.VERSION=${version}"
+  ];
+
+  meta = {
+    description = "RSS aggregator";
+    homepage = "https://github.com/mrusme/journalist";
+    license = lib.licenses.gpl3Only;
+    maintainers = with lib.maintainers; [ moraxyc ];
+    mainProgram = "journalist";
+    platforms = with lib.platforms; unix ++ windows;
+    broken = stdenv.isDarwin; # needing some apple_sdk packages
```

Comment:
Why not just add them?

Reply:
Ah…I don’t have any darwin device, this line is actually suggested by @0x61nas .
https://github.com/NixOS/nixpkgs/pull/318324#discussion_r1632130955